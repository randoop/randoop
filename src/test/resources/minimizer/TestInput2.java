package minimizer;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInput2 {
  public static boolean debug = false;

  @Test
  public void test1() throws Throwable {

    if (debug) {
      System.out.format("%n%s%n", "ErrorTest0.test01");
    }

    org.apache.commons.lang3.tuple.MutablePair[] mutablePair_array1 =
        new org.apache.commons.lang3.tuple.MutablePair[0];
    @SuppressWarnings("unchecked")
    org.apache.commons.lang3.tuple.MutablePair<
                org.apache.commons.lang3.concurrent.CircuitBreakingException,
                org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
            []
        mutablepair_circuitBreakingException_oPTION_array2 =
            (org.apache.commons.lang3.tuple.MutablePair<
                        org.apache.commons.lang3.concurrent.CircuitBreakingException,
                        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
                    [])
                mutablePair_array1;
    org.apache.commons.lang3.SerializationException serializationException4 =
        new org.apache.commons.lang3.SerializationException();
    org.apache.commons.lang3.SerializationException serializationException5 =
        org.apache.commons.lang3.SerializationUtils.clone(serializationException4);
    int i6 =
        org.apache.commons.lang3.exception.ExceptionUtils.getThrowableCount(
            (Throwable) serializationException5);
    org.apache.commons.lang3.concurrent.CircuitBreakingException circuitBreakingException7 =
        new org.apache.commons.lang3.concurrent.CircuitBreakingException(
            "hi!", (Throwable) serializationException5);
    org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION oPTION8 =
        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION.semiColonOptional;
    org.apache.commons.lang3.tuple.MutablePair<
            org.apache.commons.lang3.concurrent.CircuitBreakingException,
            org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
        mutablepair_circuitBreakingException_oPTION9 =
            org.apache.commons.lang3.tuple.MutablePair.of(circuitBreakingException7, oPTION8);
    org.apache.commons.lang3.SerializationException serializationException11 =
        new org.apache.commons.lang3.SerializationException();
    org.apache.commons.lang3.SerializationException serializationException12 =
        org.apache.commons.lang3.SerializationUtils.clone(serializationException11);
    int i13 =
        org.apache.commons.lang3.exception.ExceptionUtils.getThrowableCount(
            (Throwable) serializationException12);
    org.apache.commons.lang3.concurrent.CircuitBreakingException circuitBreakingException14 =
        new org.apache.commons.lang3.concurrent.CircuitBreakingException(
            "hi!", (Throwable) serializationException12);
    org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION oPTION15 =
        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION.semiColonOptional;
    org.apache.commons.lang3.tuple.MutablePair<
            org.apache.commons.lang3.concurrent.CircuitBreakingException,
            org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
        mutablepair_circuitBreakingException_oPTION16 =
            org.apache.commons.lang3.tuple.MutablePair.of(circuitBreakingException14, oPTION15);
    org.apache.commons.lang3.SerializationException serializationException18 =
        new org.apache.commons.lang3.SerializationException();
    org.apache.commons.lang3.SerializationException serializationException19 =
        org.apache.commons.lang3.SerializationUtils.clone(serializationException18);
    int i20 =
        org.apache.commons.lang3.exception.ExceptionUtils.getThrowableCount(
            (Throwable) serializationException19);
    org.apache.commons.lang3.concurrent.CircuitBreakingException circuitBreakingException21 =
        new org.apache.commons.lang3.concurrent.CircuitBreakingException(
            "hi!", (Throwable) serializationException19);
    org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION oPTION22 =
        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION.semiColonOptional;
    org.apache.commons.lang3.tuple.MutablePair<
            org.apache.commons.lang3.concurrent.CircuitBreakingException,
            org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
        mutablepair_circuitBreakingException_oPTION23 =
            org.apache.commons.lang3.tuple.MutablePair.of(circuitBreakingException21, oPTION22);
    org.apache.commons.lang3.SerializationException serializationException25 =
        new org.apache.commons.lang3.SerializationException();
    org.apache.commons.lang3.SerializationException serializationException26 =
        org.apache.commons.lang3.SerializationUtils.clone(serializationException25);
    int i27 =
        org.apache.commons.lang3.exception.ExceptionUtils.getThrowableCount(
            (Throwable) serializationException26);
    org.apache.commons.lang3.concurrent.CircuitBreakingException circuitBreakingException28 =
        new org.apache.commons.lang3.concurrent.CircuitBreakingException(
            "hi!", (Throwable) serializationException26);
    org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION oPTION29 =
        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION.semiColonOptional;
    org.apache.commons.lang3.tuple.MutablePair<
            org.apache.commons.lang3.concurrent.CircuitBreakingException,
            org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
        mutablepair_circuitBreakingException_oPTION30 =
            org.apache.commons.lang3.tuple.MutablePair.of(circuitBreakingException28, oPTION29);
    org.apache.commons.lang3.SerializationException serializationException32 =
        new org.apache.commons.lang3.SerializationException();
    org.apache.commons.lang3.SerializationException serializationException33 =
        org.apache.commons.lang3.SerializationUtils.clone(serializationException32);
    int i34 =
        org.apache.commons.lang3.exception.ExceptionUtils.getThrowableCount(
            (Throwable) serializationException33);
    org.apache.commons.lang3.concurrent.CircuitBreakingException circuitBreakingException35 =
        new org.apache.commons.lang3.concurrent.CircuitBreakingException(
            "hi!", (Throwable) serializationException33);
    org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION oPTION36 =
        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION.semiColonOptional;
    org.apache.commons.lang3.tuple.MutablePair<
            org.apache.commons.lang3.concurrent.CircuitBreakingException,
            org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
        mutablepair_circuitBreakingException_oPTION37 =
            org.apache.commons.lang3.tuple.MutablePair.of(circuitBreakingException35, oPTION36);
    org.apache.commons.lang3.tuple.MutablePair[] mutablePair_array39 =
        new org.apache.commons.lang3.tuple.MutablePair[5];
    @SuppressWarnings("unchecked")
    org.apache.commons.lang3.tuple.MutablePair<
                org.apache.commons.lang3.concurrent.CircuitBreakingException,
                org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
            []
        mutablepair_circuitBreakingException_oPTION_array40 =
            (org.apache.commons.lang3.tuple.MutablePair<
                        org.apache.commons.lang3.concurrent.CircuitBreakingException,
                        org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
                    [])
                mutablePair_array39;
    mutablepair_circuitBreakingException_oPTION_array40[0] =
        mutablepair_circuitBreakingException_oPTION9;
    mutablepair_circuitBreakingException_oPTION_array40[1] =
        mutablepair_circuitBreakingException_oPTION16;
    mutablepair_circuitBreakingException_oPTION_array40[2] =
        mutablepair_circuitBreakingException_oPTION23;
    mutablepair_circuitBreakingException_oPTION_array40[3] =
        mutablepair_circuitBreakingException_oPTION30;
    mutablepair_circuitBreakingException_oPTION_array40[4] =
        mutablepair_circuitBreakingException_oPTION37;
    org.apache.commons.lang3.tuple.MutablePair<
                org.apache.commons.lang3.concurrent.CircuitBreakingException,
                org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION>
            []
        mutablepair_circuitBreakingException_oPTION_array51 =
            org.apache.commons.lang3.ArrayUtils.addAll(
                mutablepair_circuitBreakingException_oPTION_array2,
                mutablepair_circuitBreakingException_oPTION_array40);

    // Checks the contract: compareTo-anti-symmetric on mutablepair_circuitBreakingException_oPTION9 and mutablepair_circuitBreakingException_oPTION23
    org.junit.Assert.assertTrue(
        "Contract failed: compareTo-anti-symmetric on mutablepair_circuitBreakingException_oPTION9 and mutablepair_circuitBreakingException_oPTION23",
        Math.signum(
                mutablepair_circuitBreakingException_oPTION9.compareTo(
                    mutablepair_circuitBreakingException_oPTION23))
            == -Math.signum(
                mutablepair_circuitBreakingException_oPTION23.compareTo(
                    mutablepair_circuitBreakingException_oPTION9)));
  }
}
