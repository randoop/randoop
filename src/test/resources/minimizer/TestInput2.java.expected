package test;

import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;
import org.apache.commons.lang3.tuple.MutablePair;
import org.apache.commons.lang3.SerializationException;
import org.apache.commons.lang3.concurrent.CircuitBreakingException;
import org.apache.commons.lang3.text.translate.NumericEntityUnescaper.OPTION;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInput2Minimized {

    public static boolean debug = false;

    @Test
    public void test1() throws Throwable {
        SerializationException serializationException5 = null;
        CircuitBreakingException circuitBreakingException7 = new CircuitBreakingException("hi!", (java.lang.Throwable) serializationException5);
        OPTION oPTION8 = null;
        MutablePair<CircuitBreakingException, OPTION> mutablepair_circuitBreakingException_oPTION9 = MutablePair.of(circuitBreakingException7, oPTION8);
        SerializationException serializationException19 = null;
        CircuitBreakingException circuitBreakingException21 = new CircuitBreakingException("hi!", (java.lang.Throwable) serializationException19);
        OPTION oPTION22 = null;
        MutablePair<CircuitBreakingException, OPTION> mutablepair_circuitBreakingException_oPTION23 = MutablePair.of(circuitBreakingException21, oPTION22);
        // Checks the contract: compareTo-anti-symmetric on mutablepair_circuitBreakingException_oPTION9 and mutablepair_circuitBreakingException_oPTION23
        org.junit.Assert.assertTrue("Contract failed: compareTo-anti-symmetric on mutablepair_circuitBreakingException_oPTION9 and mutablepair_circuitBreakingException_oPTION23", Math.signum(mutablepair_circuitBreakingException_oPTION9.compareTo(mutablepair_circuitBreakingException_oPTION23)) == -Math.signum(mutablepair_circuitBreakingException_oPTION23.compareTo(mutablepair_circuitBreakingException_oPTION9)));
    }
}
